crates/wdl-engine/src/backend.rs

```diff
@@ -148,37 +212,24 @@ impl fmt::Debug for TaskSpawnInfo {
 
 /// Represents a request to spawn a task.
 #[derive(Debug)]
-pub struct TaskSpawnRequest {
+pub(crate) struct TaskSpawnRequest {
     /// The id of the task being spawned.
     id: String,
     /// The information for the task to spawn.
     info: TaskSpawnInfo,
-    /// The attempt number for the spawn request.
-    attempt: u64,
     /// The attempt directory for the task's execution.
     attempt_dir: PathBuf,
-    /// The root directory for the evaluation.
-    task_eval_root: PathBuf,
     /// The temp directory for the evaluation.
     temp_dir: PathBuf,
 }
 
 impl TaskSpawnRequest {
     /// Creates a new task spawn request.
-    pub fn new(
-        id: String,
-        info: TaskSpawnInfo,
-        attempt: u64,
-        attempt_dir: PathBuf,
-        task_eval_root: PathBuf,
-        temp_dir: PathBuf,
-    ) -> Self {
+    pub fn new(id: String, info: TaskSpawnInfo, attempt_dir: PathBuf, temp_dir: PathBuf) -> Self {
         Self {
             id,
             info,
-            attempt,
```

Comment:
Should these removals be noted in the `CHANGELOG.md`?

Reply:
I left this as a comment to myself, and I didn't end up seeing in the notes why these were removed. So I'll leave it here.